.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.1
```

Comment:
Why do we need this?

Reply:
We didn't.  Errant check in.  I just fixed it.